src/core/filters/filterregistry.js

```diff
@@ -56,56 +57,48 @@ export default class FilterRegistry {
   }
 
   /**
-   * Gets the static filters as a {@link CombinedFilter} to send to the answers-core
+   * Gets the static filters as a {@link SimpleFilter|CombinedFilter} to send to the answers-core
+   * @returns {CombinedFilter|SimpleFilter|undefined} Returns undefined if no filters with
+   *                                                  filtering logic are present.
    */
   getStaticFilterPayload () {
     const filterNodes = this.getStaticFilterNodes();
-
-    const transformedFilterNodes = filterNodes.map(filterNode => {
-      return this._transformFilter(filterNode);
-    }).filter(filter => filter);
-
-    if (transformedFilterNodes.length > 1) {
-      return {
-        combinator: '$and',
-        filters: transformedFilterNodes
-      };
-    }
-    return transformedFilterNodes[0];
+    return this._transformFilterNodes(filterNodes, FilterCombinator.AND);
   }
 
-  _transformFilter (filterNode) {
-    if (filterNode.children) {
-      return this._recurseCombinedFilterNodes(filterNode.children, filterNode.combinator);
-    }
+  /**
+   * Transforms a filter node {@link CombinedFilterNode} or {@link SimpleFilterNode} to
+   * answers-core's {@link SimpleFilter} or {@link CombinedFilter}
+   *
+   * @param {Array<CombinedFilterNode|SimpleFilterNode>} filterNodes
+   * @param {FilterCombinator} combinator from answers-core
+   * @returns {CombinedFilter|SimpleFilter|undefined} Returns undefined if no filters with
+   *                                                  filtering logic are present.
+   */
+  _transformFilterNodes (filterNodes, combinator) {
+    const filters = filterNodes.flatMap(filterNode => {
+      if (filterNode.children) {
+        return this._transformFilterNodes(filterNode.children, filterNode.combinator);
+      }
 
-    // All static filters come through this code path, even if the filters have not been
-    // interacted with. We should ignore filters that effectively don't filter anything.
-    if (!filterNode.filter || Object.entries(filterNode.filter).length === 0) {
-      return;
-    }
+      return !this._isEmpty(filterNode) && this._transformSimpleFilterNode(filterNode);
+    }).filter(filter => filter);
```

Comment:
Since we have the `!this._isEmpty` check do we need this `filter` any more?

Reply:
Ooh, per Oliver's comment I think we can move the `isEmpty` check up to the `getStaticFilterPayload` and then yes we won't need this! Removing!